src/dbos-runtime/config.ts

```diff
@@ -181,8 +181,18 @@ export function parseConfigFile(cliOptions?: DBOSCLIStartOptions, useProxy: bool
   /*************************************/
   /* Build final runtime Configuration */
   /*************************************/
+  const entrypoints: string[] = []
+  // CLI overrides configuration
+  if (cliOptions?.entrypoint) {
```

Comment:
I think the commander package we use supports [variadic option](https://github.com/tj/commander.js?tab=readme-ov-file#variadic-option). So even with a single `--entrypoint` option we can append a list of files to it. The reason is that it'd be very confusing if our CLI `--entrypoint` only accepts one entrypoint, while our config file can specify multiple entrypoints.